cmd/otelcontribcol/go.mod

```diff
@@ -495,6 +495,7 @@ require (
 	github.com/openzipkin/zipkin-go v0.4.1 // indirect
 	github.com/outcaste-io/ristretto v0.2.0 // indirect
 	github.com/ovh/go-ovh v1.3.0 // indirect
+	github.com/panta/machineid v1.0.2 // indirect
```

Comment:
Why do we need to bring a fork dependency with 8 stars on github if we can get this info from goputil?

Reply:
@sumo-drosiek if you agree with my concern, can you please follow up with an issue or PR?

Reply:
https://github.com/panta/machineid (8 stars) is fork of https://github.com/denisbrodbeck/machineid (715 stars, but seems to be not maintained: https://github.com/denisbrodbeck/machineid/issues/5#issuecomment-752416741)

I will change it to gopsutil and we can eventually look for another solution in case it won't be sufficient

Reply:
Opened new PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18740